js/app.js

```diff
@@ -2,299 +2,100 @@
-			// append it to the DOM
-			// parent.appendChild(child);
-			ulEl.appendChild(liEl);
-		}
-		// create an element
-		var liEl = document.createElement('li');
-		// render dayTotalCookies
-		liEl.className = ('total');
-		liEl.textContent = 'Total: ' + Math.round(this.dayTotalCookies) + ' cookies';
-		// Append to DOM
-		ulEl.appendChild(liEl);
-	}
-};
+  // Create Store Location Column Header
+  thEl.textContent = 'Store Location';
+  trEl.appendChild(thEl);
 
+  // Create Hourly Time Cells
+  for (var i = 0; i < times.length; i++) {
+    thEl = document.createElement('th');
+    thEl.textContent = times[i];
+    trEl.appendChild(thEl);
+  }
+  // Create Total Sales Column Header
+  thEl = document.createElement('th');
+  thEl.textContent = 'Total Day Sales';
+  trEl.appendChild(thEl);
 
-var alki = {
-	// min and max customers per hour
-	minCustPerHour: 2,
-	maxCustPerHour: 16,	
-	avgCookPerCust: 4.6,
-	
-	// methodToGenerateRandomCustomers		
-	calcCustPerHour: function() {
-		for (var i = 0; i < times.length; i++) {
-			this.custPerHour.push(Math.floor(Math.random() * (this.maxCustPerHour - this.minCustPerHour)) + this.minCustPerHour);
-		}
-	},
-	// Array customersEachHour
-	custPerHour: [],
-	
-	// cookiesSoldEachHour = []methodToCalc
-	multiplyCookies: function() {
-		this.calcCustPerHour();
-		for (var i = 0; i < this.custPerHour.length; i++) {
-			this.cookiesPerHour.push(this.custPerHour[i] * this.avgCookPerCust);
-		}
-	},
-	// Array cookies sold per hour
-	cookiesPerHour: [],
-	
-	// TotalCookiesForTheDay = methodToCalc
-	totalCookies: function() {
-		this.multiplyCookies();
-		for (var x = 0; x < this.cookiesPerHour.length; x++) {
-			this.dayTotalCookies += this.cookiesPerHour[x];
-		}
-	},
-	dayTotalCookies: 0,
-	
-	// renderMethod
-	render: function() {
-		this.totalCookies();
-		var ulEl = document.getElementById('alki');
-		for (var i = 0; i < this.cookiesPerHour.length; i++) {
-			// create an element
-			var liEl = document.createElement('li');
-			// give it content
-			liEl.textContent = times[i] + ': ' + Math.round(this.cookiesPerHour[i]);
-			// append it to the DOM
-			// parent.appendChild(child);
-			ulEl.appendChild(liEl);
-		}
-		// create an element
-		var liEl = document.createElement('li');
-		// render dayTotalCookies
-		liEl.className = ('total');
-		liEl.textContent = 'Total: ' + Math.round(this.dayTotalCookies) + ' cookies';
-		// Append to DOM
-		ulEl.appendChild(liEl);
-	}
-};
+  // Append to Table
+  cookiesTable.append(trEl);
+}
 
+function cookieRows() {
+  headerRow();
 
-firstAndPike.render();
-seaTac.render();
-seaCenter.render();
-capHill.render();
-alki.render();
+  for (var i = 0; i < storeList.length; i++) {
+    storeList[i].render();
+  }
+}
 
+// Instantiate constructor Store() for each set of inputs
+new Store('pikePlace', 23, 65, 6.3);
+new Store('seaTac', 3, 24, 1.2);
+new Store('seaCenter', 11, 38, 3.7);
+new Store('capHill', 20, 38, 2.3);
+new Store('alki', 3, 24, 1.2);
```

Comment:
You have the incorrect numbers for the Alki store.

Reply:
Also, you can use the actual names for these locations instead of abbreviated in camel-case.